ec2net-functions

```diff
@@ -176,6 +208,11 @@ rewrite_primary() {
   fi
   logger --tag ec2net "[rewrite_primary] Rewriting configs for ${INTERFACE}"
   cidr=$(get_cidr)
+  if [ "${?}" -gt 0 ]; then
+    # For any errors from IMDS, bail out early rather than rewriting anything
+    # We'll get back here later and be able to rewrite things.
+    return $?
+  fi
   if [ -z ${cidr} ]; then
     return
   fi
```

Comment:
Is there a case where this condition will still match, after the addition above?

Reply:
I don't *think* so? I'd have to dig into ENI docs though.